### PR TITLE
Alpha guides

### PR DESCRIPTION
**Pull request by IagoLast**
https://github.com/CartoDB/carto-vl/issues/62
---

- Unify expression doc
- Add guide only for color expressions

### CODE REVIEW

File: docs/guides/01-quickstart.md [hunk 1]
```diff
@@ -23,3 +23,8 @@ CARTO GL is the perfect library to create the location intelligence apps of the
 - It allows to paint maps in client using webgl technologies
 - Enrich your maps with dynamic styles, animations and interactivity
 - Create widgets and elements to show geospatial data in different ways
+
+# Getting started
+
+First you need to create a CARTO account and get your [credentials](https://carto.com/developers/fundamentals/authorization/) remember that to use CARTO VL you need to [autenticate the client](https://carto.com/developers/carto-vl/reference/#cartosetdefaultauth) first.
```
Comment (davidmanzanares) [inline docs/guides/01-quickstart.md]: This is not true, you can use it with GeoJSON sources

Comment (IagoLast) [reply docs/guides/01-quickstart.md]: we need to focus on the use of the CARTO backend

cc @rochoa @inigomedina

**Comment by davidmanzanares** [comment]
Could you fix the conflicts, pls?

**Comment by arianaescobar** [comment]
@davidmanzanares just did

File: docs/guides/02-styling-points.md [hunk 2]
```diff
@@ -56,49 +72,31 @@ Points are clustered within the defined resolution and the placement of the poin
...
 ```
 width: 5
 color: ramp($commodity_name,Prism)
 ```
-[Example](https://cartodb.github.io/carto-vl/example/mapbox.html#eyJhIjoiY3JvcF9sb3NzXzIwMTciLCJiIjoiIiwiYyI6ImNhcnRvZ2wiLCJkIjoiaHR0cHM6Ly97dXNlcn0uY2FydG8uY29tIiwiZSI6IndpZHRoOiA1XG5jb2xvcjogcmFtcCgkY29tbW9kaXR5X25hbWUsUHJpc20pIiwiZiI6eyJsbmciOi05Ni42NTI3OTQzMzU0NTkyNSwibGF0Ijo0OC4wMDcyNDM5MjcyNjEzNn0sImciOjIuOTc3NzE5MDI5Nzg1NjczfQ==)
 
 ### Most common category
 
-To color features by the most common category, you can use a metadata operatoror an aggregation operator.
+To color features by the most common category, you can use a metadata operator an aggregation operator.
```
Comment (davidmanzanares) [inline docs/guides/02-styling-points.md]: I wouldn't use the term `operator`

File: docs/guides/02-styling-points.md [hunk 3]
```diff
@@ -56,49 +72,31 @@ Points are clustered within the defined resolution and the placement of the poin
...
 
 #### Metadata Operator
 The map below uses the metadata operator `top` to retrieve the top 4 values in the `commodity_name` field. Each category is colored with a unique color and all other features are colored as other.
```
Comment (davidmanzanares) [inline docs/guides/02-styling-points.md]: This text doesn't make sense anymore without the example

File: docs/guides/02-styling-points.md [hunk 4]
```diff
@@ -56,49 +72,31 @@ Points are clustered within the defined resolution and the placement of the poin
...
 
-```
-width: 5
-color: ramp(top($commodity_name,5),Prism)
-```
-[Example](https://cartodb.github.io/carto-vl/example/mapbox.html#eyJhIjoiY3JvcF9sb3NzXzIwMTciLCJiIjoiIiwiYyI6ImNhcnRvZ2wiLCJkIjoiaHR0cHM6Ly97dXNlcn0uY2FydG8uY29tIiwiZSI6IndpZHRoOiA1XG5jb2xvcjogcmFtcCh0b3AoJGNvbW1vZGl0eV9uYW1lLDUpLFByaXNtKVxuIiwiZiI6eyJsbmciOi05Ni4xMTY5MjM2NzkzNDc3OSwibGF0Ijo0Ny41NTIxNjA5MDU5OTIxMX0sImciOjMuMDY4NDU0MTIwMzM3NjI4OH0=)
+### Resolution
+Use the `resolution` property to take advantage of the potential of [CARTO aggregations](TODO).
```
Comment (davidmanzanares) [inline docs/guides/02-styling-points.md]: TODO

File: docs/guides/03-styling-lines.md [hunk 5]
```diff
@@ -4,31 +4,30 @@ This guide walks through making a variety of common thematic map types with line
...
+### Color
+Use the `color` property to specify the line color with one of CARTO VL's [supported color expressions](LINK).
 
 ```
 width: 1.5
 color: hsv(0.5,1,1)
 ```
 
-### Line Draw Order
+TODO: REVIEW FROM HERE
```
Comment (davidmanzanares) [inline docs/guides/03-styling-lines.md]: TODO

File: docs/guides/03-styling-lines.md [hunk 6]
```diff
@@ -112,4 +111,4 @@ color: ramp(buckets($price,50,100,500,1000),sunset)
 ```
 
 ### Manually Classed Size
-TBD 
\ No newline at end of file
+TBD
```
Comment (davidmanzanares) [inline docs/guides/03-styling-lines.md]: TBD

**Comment by arianaescobar** [comment]
Hmm, I think it might be better if I start a new branch instead...

**Comment by arianaescobar** [comment]
Yeah. My bad. I started a new branch. Sorry guys

**Comment by IagoLast** [comment]
outdated

**Pull request closed without merging**